### PR TITLE
Attempting to split the message courses.active_courses for issue #4342

### PR DESCRIPTION
**Pull request by Sahethi**
NOTE: Please review the pull request process before opening your first PR: https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/CONTRIBUTING.md#pull-request-process

## What this PR does
This PR is trying to solve the issue #4342 
In **config/locales/en.yml** the value was changed 
to `active_courses: Active Courses` 
from `active_courses: Active %{course_type}` where the `%{course_type}` was referencing `courses: Courses`  hence didn't serve the purpose, hence now the action is similar to what active_campaign does.

## Screenshot
Element dealt with
image

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # of editing this file, please use the migrations feature of Active Record to
 # incrementally modify your database, and then regenerate this schema definition.
 #
-# This file is the source Rails uses to define your schema when running `rails
-# db:schema:load`. When creating a new database, `rails db:schema:load` tends to
+# This file is the source Rails uses to define your schema when running `bin/rails
```
Comment (ragesoss) [inline db/schema.rb:5]: Please exclude the changes to schema.rb from this PR.

Comment (Sahethi) [reply db/schema.rb]: Hello, I have excluded the changes in **db/schema.rb** and also changed the **qqq.yml** file.

File: config/locales/qqq.yml [hunk 2]
```diff
@@ -242,6 +242,10 @@ qqq:
     milestones:
       title: Label for the Milestone section of a course page (Wiki Ed only)
   campaign:
+    active_campaigns: |-
+      Heading for the list of active campaigns.
```
Comment (ragesoss) [inline config/locales/qqq.yml:246]: This should be just "Heading for the list of active campaigns". The rest can be removed.

File: config/locales/qqq.yml [hunk 3]
```diff
@@ -296,11 +300,8 @@ qqq:
   courses:
     activity: Label for link to the 'recent activity' list for a course
     active_courses: |-
-      Heading for the list of active courses. 'course_type' is one of these messages:
-      * {{msg-wm|Wikiedudashboard-courses.courses}}
-      * {{msg-wm|Wikiedudashboard-campaign.campaigns}}
-
-      So in English, the message would say either 'Active Courses' or 'Active Campaigns'.
+      Heading for the list of active courses.
```
[the remark this answers is not in the document]
Comment (ragesoss) [reply config/locales/qqq.yml:303]: Yes. You can also omit the period at the end, to match the convention in the rest of the file.

[the remark this answers is not in the document]
Comment (Sahethi) [reply config/locales/qqq.yml]: Hey I have made the necessary changes.

File: config/locales/qqq.yml [hunk 4]
```diff
@@ -242,6 +242,9 @@ qqq:
     milestones:
       title: Label for the Milestone section of a course page (Wiki Ed only)
   campaign:
+    active_campaigns: |-
+      Heading for the list of active campaigns
+      {{Identical|Active}}
```
Comment (ragesoss) [inline config/locales/qqq.yml:247]: This line is not needed

File: config/locales/qqq.yml [hunk 5]
```diff
@@ -296,11 +299,7 @@ qqq:
...
     active_courses: |-
-      Heading for the list of active courses. 'course_type' is one of these messages:
-      * {{msg-wm|Wikiedudashboard-courses.courses}}
-      * {{msg-wm|Wikiedudashboard-campaign.campaigns}}
-
-      So in English, the message would say either 'Active Courses' or 'Active Campaigns'.
+      Heading for the list of active courses
       {{Identical|Active}}
     all_courses: Button to view the list of all courses
     already_enrolled: Message displayed when the user attempts to enroll in a course
```
Comment (ragesoss) [inline config/locales/qqq.yml:305]: Line 303 here is not needed.

File: db/schema.rb [hunk 6]
```diff
@@ -595,4 +595,4 @@
   end
 
   add_foreign_key "course_stats", "courses"
-end
+end
```
Comment (ragesoss) [inline db/schema.rb:598]: Ideally, this should not be part of the changeset.

Comment (Sahethi) [reply db/schema.rb:598]: I'm not sure why my db.schema gets updated, cause I didn't perform any changes in it essentially. It seemed to have gotten updated when I launched the server.

Comment (ragesoss) [reply db/schema.rb:598]: In this case, it's just whether there's a final newline at the end of the file, which is something that usually happens automatically (or not) and varies by operating system. This change doesn't actually matter, but I'd rather have a clean PR.

File: config/locales/qqq.yml (hunk 5 above)
Comment (Sahethi) [reply config/locales/qqq.yml:303]: Just fixed it.

File: db/schema.rb (hunk 6 above)
Comment (Sahethi) [reply db/schema.rb:598]: Sure I completely understand, I did switch my operating system cause I running into errors and was having trouble setting up the dev environment initially on M1 but I'll make sure that it doesn't happen again.

**Comment by Sahethi** [comment]
@ragesoss Hey, are there anymore changes required or rather should I submit a clean PR?

**Pull request closed without merging**